### DISCUSSION TITLE
Can I use function which return panda style ?

### DISCUSSION THREAD

**Discussion by azoriusss** [chakra-ui/panda#1763] [Q&A] [answered]
Hi

Is there a way to use a function to return panda style ?
Like in [this exemple](https://play.panda-css.com/68vfpp3xNu)
It seem that class are well generated but with no matching style

I found this [discussion](https://github.com/chakra-ui/panda/discussions/1540) but i'm not sure it's related

Am I missing something here ?  

Thanks 🙏

**Comment by segunadebayo** [comment, marked as answer]
Yes, you can @vlrd-playin.

Use the `css.raw()` function which is an identity function but serves as a hint to the compiler to extract styles.

![CleanShot 2023-12-04 at 16 24 01@2x](https://github.com/chakra-ui/panda/assets/6916170/d11e0f4e-f868-4bb8-9bdc-f280d253ce81)
